docs/docs.css

```diff
@@ -83,7 +83,7 @@ pre {
 }
 
 .Markdown p {
-  background-color: var(--g-colorGray0);
+  background-color: transparent;
```

Comment:
`--color-transparent`. xp

Reply:
Where are these defined? I was trying to find them

Reply:
global CSS that comes from ColorSchemeProvider:
![Screen Shot 2023-04-04 at 3 30 46 PM](https://user-images.githubusercontent.com/12059539/229936840-c6885381-c744-4239-ba5a-cdcb361d1c53.png)

if you grep for `gestaltTheme` you'll find where that class name is being set

Reply:
I was looking at the transparent, I think regular transparent is probably okay for us here